patroni/postgresql/__init__.py

```diff
@@ -200,7 +200,7 @@ def set_pending_restart(self, value):
 
     @property
     def sysid(self):
-        if not self._sysid and not self.bootstrapping:
+        if not self._sysid and not self.bootstrapping or not self.is_running():
```

Comment:
Against which situation does the `or` clause protect ?

Reply:
I'll look at it one more time tomorrow morning.